### DISCUSSION TITLE
Hide logs

### DISCUSSION THREAD

**Discussion by Compositr** [Elius94/console-gui-tools#64] [Q&A] [answered]
I am using this to build a small console application for a personal project. Is there a way to hide the logs pane?

**Comment by Elius94** [comment, marked as answer]
Hello! Yes, you can hide the logs:

add the property: options.logLocation

`
Choose where the logs are displayed: number (0,1) - to pot them on one of the two layouts, string ("popup") - to put them on a CustomPopup that can be displayed on the window.
`

Set it to "popup" and you will move the logs to a modal window that appears only by pressing a key.
The key to show log popup is `options.showLogKey`.

Let me know if it's allright!

**Comment by Compositr** [comment]
Hey, thanks for the fast response!

It seems to work but now there is just a blank bottom section (where the logs used to appear):
![image](https://github.com/Elius94/console-gui-tools/assets/43405050/edcc8a00-e163-4baa-b5fc-e2f9dbfbeda6)

**Comment by Elius94** [comment]
Yes, that's because you are using the "double" layout. (read the README
file in the layout section)
You can change it to a single layout to keep only the main box.

**Reply by Elius94** [reply]
https://github.com/Elius94/console-gui-tools#layout

**Comment by Compositr** [comment]
Worked, thanks for the help!

**Reply by Elius94** [reply]
You are welcome! thanks for using the lib
